### ISSUE TITLE
WireIt doesn't kill `ng serve` properly

### ISSUE THREAD

**Issue by SanderElias** [google/wireit#540]
This the follow-up on the discussion in #33 

I think your guess is right, and the problem is that WireIt doesn't seem able to kill `ng serve` properly.
If you look at this screenshot:
![image](https://user-images.githubusercontent.com/1249083/201902366-ca1b9d76-2d2d-46c5-bdc4-19f93eb759cd.png)

This happened after I killed WireIt Manually, and restarted the watch mode.
I `^c`'d it, and use `pkill` to kill the `ng serve` command. That resulted in 2 lines that seems to originate from WireIt.

**Comment by SanderElias** [comment]
@aomarks Still happening. 
Still feel it would be better when there is an option to kill the service (and wait for the exit) before starting to build the dependencies.

**Comment by SanderElias** [comment]
As promised in #568 I created a small repo to demo the issue I have.
steps to reproduce:

```bash
git clone git@github.com:SanderElias/wireiProblemRepo.git
cd wireiProblemRepo
npm i
npm run start
```

That should bring up the `ng serve` of the demo app. you can open this on `http://localhost:4200/`

Now open the file `projects/lib-a/src/lib/lib-a.service.ts` and change the string or something.
In most cases, the `ng serve` command will _not_ be killed at all. 
When you repeat this a couple of times, you probably will notice a build error from ng-serve that seems to stem from out-of-order execution. 

It would help to have the option to kill `ng serve` before the libs get compiled, but that is a separate issue.
